### ISSUE TITLE
CDR version

### ISSUE THREAD

**Issue by Cui-yaowen** [Veronicium/Eider#4]
What if I want to perform an experiment on the CDR dataset, can you provide the code for the CDR version?  Thanks!

**Comment by ZhichaoDuan** [comment]
Hi, have you managed to get code ?

**Comment by Cui-yaowen** [comment]
No, I haven't got the code for the CDR training, but I really need the CDR code

**Comment by ZhichaoDuan** [comment]
me too -_-

**Comment by Cui-yaowen** [comment]
So what should we do?

**Comment by passengeryjy** [comment]
Hi，have you manage to get code for cdr/gda? i will be very grateful if you can share it.

**Comment by SevenPwZ** [comment]
Hi，hava you managed to get code? I will be very grateful if you can provide the code for the CDR version. E-mail: 1481770375@qq.com
